src/async-components/views/dialogs/security/CreateSecretStorageDialog.tsx

```diff
@@ -739,7 +740,18 @@ export default class CreateSecretStorageDialog extends React.PureComponent<IProp
                             onClick={this.onCopyClick}
                             disabled={this.state.phase === Phase.Storing}
                         >
-                            { this.state.copied ? _t("Copied!") : _t("Copy") }
+                            <span 
```

Comment:
The basic idea here is hiding the text by setting its height to 0.    
This prevents the buttons from jumping around if the text changes.